src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
-body {
-  margin: 0;
+@import "./_variables/variables";
+@import "./_variables/reset";
+@import "./_variables/functions";
+
+.container {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 7 * ($size + $gap) - $gap + 2 * $padding;
```

Comment:
if you know how to use `scss functions` you cane move this calculation there

Reply:
done